JitsiConference.js

```diff
@@ -703,6 +698,11 @@ JitsiConference.prototype.leave = function() {
                     participant => this.onMemberLeft(participant.getJid()));
 
                 throw error;
+            })
+            .then(() => {
+                if (this.rtc) {
```

Comment:
how does that make sure RTC is always destroyed? it will not be in the case of the catch above because it throws at the end, am I missing something?

Reply:
Damn you're right, it slipped my mind.

Reply:
https://github.com/jitsi/lib-jitsi-meet/pull/1808